apps/libs/actuator/src/ActuatorModule.ts

```diff
@@ -1,6 +1,8 @@
 import { HealthCheckError, HealthIndicator, HealthIndicatorResult, TerminusModule } from '@nestjs/terminus'
 import { Global, Module } from '@nestjs/common'
 
+export { HealthIndicatorResult, HealthCheckError }
```

Comment:
Should separate this into another PR.

Reply:
yea i just removed it since its from @nestjs/terminus

changes:
https://github.com/DeFiCh/jellyfish/pull/1140/files#r817350854